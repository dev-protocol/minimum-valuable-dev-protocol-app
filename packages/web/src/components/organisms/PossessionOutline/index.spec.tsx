import React from 'react'
import { render } from '@testing-library/react'
import { PossessionOutline } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/dev-kit/hooks')
jest.mock('src/fixtures/utility/gql-hooks-wrapper')

describe(`${PossessionOutline.name}`, () => {
  test('Snapshot', () => {
    const component = render(<PossessionOutline propertyAddress="propertyAddress" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
