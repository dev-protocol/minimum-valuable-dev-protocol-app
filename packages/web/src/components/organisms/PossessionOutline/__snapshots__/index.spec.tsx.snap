// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PossessionOutline Snapshot 1`] = `
<body>
  <div>
    <div
      class="zlzook-0 bruRoN"
    >
      <div
        class="ant-card ant-card-bordered"
      >
        <div
          class="ant-card-body"
        >
          <div
            class="ant-statistic"
          >
            <div
              class="ant-statistic-title"
            >
              Total Staking Amount
            </div>
            <div
              class="ant-statistic-content"
            >
              <span
                class="ant-statistic-content-value"
              >
                <span
                  class="ant-statistic-content-value-int"
                >
                  
                  10,000
                </span>
                <span
                  class="ant-statistic-content-value-decimal"
                >
                  .00
                </span>
              </span>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-card ant-card-bordered"
      >
        <div
          class="ant-card-body"
        >
          <div
            class="ant-statistic"
          >
            <div
              class="ant-statistic-title"
            >
              Total Rewards
            </div>
            <div
              class="ant-statistic-content"
            >
              <span
                class="ant-statistic-content-value"
              >
                <span
                  class="ant-statistic-content-value-int"
                >
                  
                  10,000
                </span>
                <span
                  class="ant-statistic-content-value-decimal"
                >
                  .00
                </span>
              </span>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-card ant-card-bordered"
      >
        <div
          class="ant-card-body"
        >
          <div
            class="ant-statistic"
          >
            <div
              class="ant-statistic-title"
            >
              Your Staking Share
            </div>
            <div
              class="ant-statistic-content"
            >
              <span
                class="ant-statistic-content-value"
              >
                <span
                  class="ant-statistic-content-value-int"
                >
                  
                  50
                </span>
                <span
                  class="ant-statistic-content-value-decimal"
                >
                  .00
                </span>
              </span>
              <span
                class="ant-statistic-content-suffix"
              >
                %
              </span>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-card ant-card-bordered"
      >
        <div
          class="ant-card-body"
        >
          <div
            class="ant-statistic"
          >
            <div
              class="ant-statistic-title"
            >
              Your Staking Amount
            </div>
            <div
              class="ant-statistic-content"
            >
              <span
                class="ant-statistic-content-value"
              >
                <span
                  class="ant-statistic-content-value-int"
                >
                  
                  5,000
                </span>
                <span
                  class="ant-statistic-content-value-decimal"
                >
                  .00
                </span>
              </span>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-card ant-card-bordered"
      >
        <div
          class="ant-card-body"
        >
          <div
            class="ant-statistic"
          >
            <div
              class="ant-statistic-title"
            >
              Your Rewards
            </div>
            <div
              class="ant-statistic-content"
            >
              <span
                class="ant-statistic-content-value"
              >
                <span
                  class="ant-statistic-content-value-int"
                >
                  
                  5,000
                </span>
                <span
                  class="ant-statistic-content-value-decimal"
                >
                  .00
                </span>
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
